mix.exs

```diff
@@ -45,24 +45,23 @@ defmodule BlockScout.Mixfile do
     ]
   end
 
-  defp aliases(env) do
-    [
-      # to match behavior of `mix test` in `apps/indexer`, which needs to not start applications for `indexer` to
-      # prevent its supervision tree from starting, which is undesirable in test
-      test: "test --no-start",
-      data_migrate: "ecto.migrate --migrations-path=apps/explorer/priv/repo/data_migrations -n 1 --log-sql"
-    ] ++ env_aliases(env)
-  end
+  # defp aliases(env) do
+  #   [
+  #     # to match behavior of `mix test` in `apps/indexer`, which needs to not start applications for `indexer` to
+  #     # prevent its supervision tree from starting, which is undesirable in test
+  #     test: "test --no-start"
+  #   ] ++ env_aliases(env)
+  # end
 
-  defp env_aliases(:dev) do
-    []
-  end
+  # defp env_aliases(:dev) do
+  #   []
+  # end
 
-  defp env_aliases(_env) do
-    [
-      compile: "compile"
-    ]
-  end
+  # defp env_aliases(_env) do
+  #   [
+  #     compile: "compile --warnings-as-errors"
```

Comment:
upstream doesn't like `--warnings-as-errors`... does this frustrate our efforts to activate that compile option?

Reply:
Not really, we had `compile: "compile"` here anyway, we enforce it on the [build step](https://github.com/celo-org/blockscout/blob/master/.github/workflows/config.yml#L363).